[
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11735",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11735/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11735/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11735/events",
    "html_url": "https://github.com/npm/npm/issues/11735",
    "id": 137612916,
    "number": 11735,
    "title": "Installing devDependencies does not update shrinkwrap",
    "user": {
      "login": "sheepsteak",
      "id": 1379888,
      "avatar_url": "https://avatars.githubusercontent.com/u/1379888?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/sheepsteak",
      "html_url": "https://github.com/sheepsteak",
      "followers_url": "https://api.github.com/users/sheepsteak/followers",
      "following_url": "https://api.github.com/users/sheepsteak/following{/other_user}",
      "gists_url": "https://api.github.com/users/sheepsteak/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/sheepsteak/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/sheepsteak/subscriptions",
      "organizations_url": "https://api.github.com/users/sheepsteak/orgs",
      "repos_url": "https://api.github.com/users/sheepsteak/repos",
      "events_url": "https://api.github.com/users/sheepsteak/events{/privacy}",
      "received_events_url": "https://api.github.com/users/sheepsteak/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T15:59:21Z",
    "updated_at": "2016-03-01T15:59:53Z",
    "closed_at": null,
    "body": "I've created a shrink-wrap using `npm shrinkwrap --dev`. Installing new dependencies (`npm i -S express`) correctly updates the `npm-shrinkwrap.json`. However, installing a devDependency (`npm i -D eslint@1`) does **not** update the `npm-shrinkwrap.json`.\r\n\r\nThis seems counterintuitive as we've specified we want everything shrinkwrapped so you would expect `npm-shrinkwrap.json` to always be updated. To get around this at the moment we have to always remember to do `npm shrinkwrap --dev` after installing a new devDependency. Also, when updating a dependency with something like `npm i -D eslint@2` the `npm-shrinkwrap.json` is not updated either, which is probably even worse as it's already specified in the `npm-shrinkwrap.json`! :fearful:\r\n\r\nThis is with `npm@3.8.0`."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11734",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11734/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11734/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11734/events",
    "html_url": "https://github.com/npm/npm/issues/11734",
    "id": 137612511,
    "number": 11734,
    "title": "npm ERR! Darwin 14.5.0 (cant install plugins or anything)",
    "user": {
      "login": "rav-sohal",
      "id": 15650944,
      "avatar_url": "https://avatars.githubusercontent.com/u/15650944?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/rav-sohal",
      "html_url": "https://github.com/rav-sohal",
      "followers_url": "https://api.github.com/users/rav-sohal/followers",
      "following_url": "https://api.github.com/users/rav-sohal/following{/other_user}",
      "gists_url": "https://api.github.com/users/rav-sohal/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/rav-sohal/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/rav-sohal/subscriptions",
      "organizations_url": "https://api.github.com/users/rav-sohal/orgs",
      "repos_url": "https://api.github.com/users/rav-sohal/repos",
      "events_url": "https://api.github.com/users/rav-sohal/events{/privacy}",
      "received_events_url": "https://api.github.com/users/rav-sohal/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T15:58:01Z",
    "updated_at": "2016-03-01T15:58:01Z",
    "closed_at": null,
    "body": "I am getting this error:\r\n\r\nnpm ERR! Darwin 14.5.0\r\nnpm ERR! argv \"/usr/local/bin/node\" \"/usr/local/bin/npm\" \"uninstall\" \"npm\" \"-g\"\r\nnpm ERR! node v5.7.0\r\nnpm ERR! npm  v3.6.0\r\nnpm ERR! code MODULE_NOT_FOUND\r\n\r\nnpm ERR! Cannot find module 'github-url-from-git'\r\nnpm ERR! \r\nnpm ERR! If you need help, you may report this error at:\r\nnpm ERR!     <https://github.com/npm/npm/issues>\r\n\r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!     /Users/ravs/npm-debug.log\r\n\r\n\r\nAlso i can't install no plugins for cordova etc...\r\n\r\n\r\nAny ideas?"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11733",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11733/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11733/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11733/events",
    "html_url": "https://github.com/npm/npm/issues/11733",
    "id": 137594364,
    "number": 11733,
    "title": "please，help，why  error Windows_NT 10.0.10586",
    "user": {
      "login": "showerychn",
      "id": 5378579,
      "avatar_url": "https://avatars.githubusercontent.com/u/5378579?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/showerychn",
      "html_url": "https://github.com/showerychn",
      "followers_url": "https://api.github.com/users/showerychn/followers",
      "following_url": "https://api.github.com/users/showerychn/following{/other_user}",
      "gists_url": "https://api.github.com/users/showerychn/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/showerychn/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/showerychn/subscriptions",
      "organizations_url": "https://api.github.com/users/showerychn/orgs",
      "repos_url": "https://api.github.com/users/showerychn/repos",
      "events_url": "https://api.github.com/users/showerychn/events{/privacy}",
      "received_events_url": "https://api.github.com/users/showerychn/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T14:51:04Z",
    "updated_at": "2016-03-01T14:51:04Z",
    "closed_at": null,
    "body": "0 info it worked if it ends with ok\r\n1 verbose cli [ 'C:\\\\Program Files\\\\nodejs\\\\node.exe',\r\n1 verbose cli   'C:\\\\Users\\\\Administrator.2013-20150705MU\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\npm\\\\bin\\\\npm-cli.js',\r\n1 verbose cli   'run',\r\n1 verbose cli   'dev' ]\r\n2 info using npm@3.7.5\r\n3 info using node@v5.7.0\r\n4 verbose run-script [ 'predev', 'dev', 'postdev' ]\r\n5 info lifecycle sogu-fe@0.0.1~predev: sogu-fe@0.0.1\r\n6 silly lifecycle sogu-fe@0.0.1~predev: no script for predev, continuing\r\n7 info lifecycle sogu-fe@0.0.1~dev: sogu-fe@0.0.1\r\n8 verbose lifecycle sogu-fe@0.0.1~dev: unsafe-perm in lifecycle true\r\n9 verbose lifecycle sogu-fe@0.0.1~dev: PATH: C:\\Users\\Administrator.2013-20150705MU\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\node-gyp-bin;E:\\sogu-fe\\node_modules\\.bin;C:\\Program Files\\nodejs;C:\\Ruby22\\bin;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files (x86)\\Windows7Master;C:\\Program Files\\TortoiseGit\\bin;C:\\Program Files\\nodejs\\;C:\\WINDOWS\\system32;C:\\WINDOWS;C:\\WINDOWS\\System32\\Wbem;C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files (x86)\\ATI Technologies\\ATI.ACE\\Core-Static;D:\\Program Files (x86)\\nodeJs\\node_modules\\npm;C:\\Program Files\\nodejs;C:\\Users\\Administrator.2013-20150705MU\\AppData\\Roaming\\npm;C:\\Program Files (x86)\\Git\\bin;C:\\Program Files (x86)\\Git\\cmd;D:\\software\\rubygems-2.5.0\\\r\n10 verbose lifecycle sogu-fe@0.0.1~dev: CWD: E:\\sogu-fe\r\n11 silly lifecycle sogu-fe@0.0.1~dev: Args: [ '/d /s /c', 'better-npm-run dev' ]\r\n12 silly lifecycle sogu-fe@0.0.1~dev: Returned: code: 1  signal: null\r\n13 info lifecycle sogu-fe@0.0.1~dev: Failed to exec dev script\r\n14 verbose stack Error: sogu-fe@0.0.1 dev: `better-npm-run dev`\r\n14 verbose stack Exit status 1\r\n14 verbose stack     at EventEmitter.<anonymous> (C:\\Users\\Administrator.2013-20150705MU\\AppData\\Roaming\\npm\\node_modules\\npm\\lib\\utils\\lifecycle.js:239:16)\r\n14 verbose stack     at emitTwo (events.js:100:13)\r\n14 verbose stack     at EventEmitter.emit (events.js:185:7)\r\n14 verbose stack     at ChildProcess.<anonymous> (C:\\Users\\Administrator.2013-20150705MU\\AppData\\Roaming\\npm\\node_modules\\npm\\lib\\utils\\spawn.js:24:14)\r\n14 verbose stack     at emitTwo (events.js:100:13)\r\n14 verbose stack     at ChildProcess.emit (events.js:185:7)\r\n14 verbose stack     at maybeClose (internal/child_process.js:827:16)\r\n14 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)\r\n15 verbose pkgid sogu-fe@0.0.1\r\n16 verbose cwd E:\\sogu-fe\r\n\r\n17 error Windows_NT 10.0.10586\r\n18 error argv \"C:\\\\Program Files\\\\nodejs\\\\node.exe\" \"C:\\\\Users\\\\Administrator.2013-20150705MU\\\\AppData\\\\Roaming\\\\npm\\\\node_modules\\\\npm\\\\bin\\\\npm-cli.js\" \"run\" \"dev\"\r\n\r\n19 error node v5.7.0\r\n20 error npm  v3.7.5\r\n21 error code ELIFECYCLE\r\n22 error sogu-fe@0.0.1 dev: `better-npm-run dev`\r\n22 error Exit status 1\r\n23 error Failed at the sogu-fe@0.0.1 dev script 'better-npm-run dev'.\r\n23 error Make sure you have the latest version of node.js and npm installed.\r\n23 error If you do, this is most likely a problem with the sogu-fe package,\r\n23 error not with npm itself.\r\n23 error Tell the author that this fails on your system:\r\n23 error     better-npm-run dev\r\n23 error You can get information on how to open an issue for this project with:\r\n23 error     npm bugs sogu-fe\r\n23 error Or if that isn't available, you can get their info via:\r\n23 error     npm owner ls sogu-fe\r\n23 error There is likely additional logging output above.\r\n24 verbose exit [ 1, true ]\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11732",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11732/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11732/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11732/events",
    "html_url": "https://github.com/npm/npm/issues/11732",
    "id": 137565018,
    "number": 11732,
    "title": "Cat does't work with windows wild cards",
    "user": {
      "login": "DennisNerush",
      "id": 2686088,
      "avatar_url": "https://avatars.githubusercontent.com/u/2686088?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/DennisNerush",
      "html_url": "https://github.com/DennisNerush",
      "followers_url": "https://api.github.com/users/DennisNerush/followers",
      "following_url": "https://api.github.com/users/DennisNerush/following{/other_user}",
      "gists_url": "https://api.github.com/users/DennisNerush/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/DennisNerush/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/DennisNerush/subscriptions",
      "organizations_url": "https://api.github.com/users/DennisNerush/orgs",
      "repos_url": "https://api.github.com/users/DennisNerush/repos",
      "events_url": "https://api.github.com/users/DennisNerush/events{/privacy}",
      "received_events_url": "https://api.github.com/users/DennisNerush/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T12:44:41Z",
    "updated_at": "2016-03-01T12:44:41Z",
    "closed_at": null,
    "body": "I have the following file structure:\r\n\r\n|-package.json\r\n|-output/\r\n|-Client/\r\n|------ js/\r\n|-------- component1/\r\n|---------------------------one.css \r\n|---------------------------one.js \r\n|---------component2/\r\n|---------------------------two.css \r\n|---------------------------two.js \r\n\r\nThe package.json and Client directory are in the same root directory.\r\nI want to cat all the css files into a single css file.\r\n\r\nHere is my Package.json:\r\n  \"scripts\": {\r\n    \t\t\"cat-css\": \"cat Client/js/*.css > output/styles.css\"\r\n  },\r\n\r\nWhen I run npm run cat-css I get the following error:\r\n\r\nError: ENOENT: no such file or directory, open 'C:\\dev\\somefolder\\somefolder\\somefolder\\Client\\js\\*.css'\r\n    at Error (native)\r\n\r\nI'm ruining it on Windows. Perhaps there is something wrong with the wild card?\r\n\r\n[npm-debug.txt](https://github.com/npm/npm/files/152765/npm-debug.txt)\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11731",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11731/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11731/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11731/events",
    "html_url": "https://github.com/npm/npm/issues/11731",
    "id": 137552835,
    "number": 11731,
    "title": "Property with the same name already exists: Cannot add property dist",
    "user": {
      "login": "zalow517",
      "id": 4902549,
      "avatar_url": "https://avatars.githubusercontent.com/u/4902549?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/zalow517",
      "html_url": "https://github.com/zalow517",
      "followers_url": "https://api.github.com/users/zalow517/followers",
      "following_url": "https://api.github.com/users/zalow517/following{/other_user}",
      "gists_url": "https://api.github.com/users/zalow517/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/zalow517/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/zalow517/subscriptions",
      "organizations_url": "https://api.github.com/users/zalow517/orgs",
      "repos_url": "https://api.github.com/users/zalow517/repos",
      "events_url": "https://api.github.com/users/zalow517/events{/privacy}",
      "received_events_url": "https://api.github.com/users/zalow517/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T11:52:09Z",
    "updated_at": "2016-03-01T11:53:26Z",
    "closed_at": null,
    "body": "Hi all,\r\n\r\nI am encountering issues with installing a private package after publishing it successfully. I can't understand what's going on here and I've tried running the same thing on a different machine with the same Node/NPM setup.\r\n\r\nThe first time I published this package, everything worked fine - then I updated the package and republished. Now not even the first package that I had published is working, and is showing me the same error as the below:\r\n\r\n29 info retry will retry, error on last attempt: Error: ArgumentException Can not add property dist to Newtonsoft.Json.Linq.JObject. Property with the same name already exists on object.: npm\r\n30 info attempt registry request try #2 at 12:29:59 PM\r\n31 http request GET <private repo URL>\r\n32 http 500 <private repo URL>\r\n33 verbose headers { 'cache-control': 'private',\r\n33 verbose headers   'content-length': '153',\r\n33 verbose headers   'content-type': 'application/json; charset=utf-8',\r\n33 verbose headers   server: 'Microsoft-IIS/8.5',\r\n33 verbose headers   'x-aspnet-version': '4.0.30319',\r\n33 verbose headers   'x-powered-by': 'ASP.NET',\r\n33 verbose headers   date: 'Tue, 01 Mar 2016 11:29:59 GMT' }\r\n\r\nAny ideas? Tried searching everywhere but the E500 error code seems to be a very generic one and couldn't find anything related to 'npm install' for it."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11730",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11730/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11730/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11730/events",
    "html_url": "https://github.com/npm/npm/issues/11730",
    "id": 137524755,
    "number": 11730,
    "title": "Firebase is not getting installed - npm",
    "user": {
      "login": "AshayAgrawal",
      "id": 1709933,
      "avatar_url": "https://avatars.githubusercontent.com/u/1709933?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/AshayAgrawal",
      "html_url": "https://github.com/AshayAgrawal",
      "followers_url": "https://api.github.com/users/AshayAgrawal/followers",
      "following_url": "https://api.github.com/users/AshayAgrawal/following{/other_user}",
      "gists_url": "https://api.github.com/users/AshayAgrawal/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/AshayAgrawal/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/AshayAgrawal/subscriptions",
      "organizations_url": "https://api.github.com/users/AshayAgrawal/orgs",
      "repos_url": "https://api.github.com/users/AshayAgrawal/repos",
      "events_url": "https://api.github.com/users/AshayAgrawal/events{/privacy}",
      "received_events_url": "https://api.github.com/users/AshayAgrawal/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T09:52:06Z",
    "updated_at": "2016-03-01T09:52:06Z",
    "closed_at": null,
    "body": "Hi,\r\n\r\nI am not able to install firebase tools using npm. Logs are attached. Please help fix.\r\n[npm-debug.txt](https://github.com/npm/npm/files/152500/npm-debug.txt)\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11729",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11729/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11729/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11729/events",
    "html_url": "https://github.com/npm/npm/pull/11729",
    "id": 137517961,
    "number": 11729,
    "title": "Fix #11003 remove-verison-alias-from-cli",
    "user": {
      "login": "danielkman",
      "id": 2141074,
      "avatar_url": "https://avatars.githubusercontent.com/u/2141074?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/danielkman",
      "html_url": "https://github.com/danielkman",
      "followers_url": "https://api.github.com/users/danielkman/followers",
      "following_url": "https://api.github.com/users/danielkman/following{/other_user}",
      "gists_url": "https://api.github.com/users/danielkman/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/danielkman/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/danielkman/subscriptions",
      "organizations_url": "https://api.github.com/users/danielkman/orgs",
      "repos_url": "https://api.github.com/users/danielkman/repos",
      "events_url": "https://api.github.com/users/danielkman/events{/privacy}",
      "received_events_url": "https://api.github.com/users/danielkman/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [
      {
        "url": "https://api.github.com/repos/npm/npm/labels/review",
        "name": "review",
        "color": "343434"
      }
    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T09:19:19Z",
    "updated_at": "2016-03-01T09:19:23Z",
    "closed_at": null,
    "pull_request": {
      "url": "https://api.github.com/repos/npm/npm/pulls/11729",
      "html_url": "https://github.com/npm/npm/pull/11729",
      "diff_url": "https://github.com/npm/npm/pull/11729.diff",
      "patch_url": "https://github.com/npm/npm/pull/11729.patch"
    },
    "body": "I also saw `isntall` alias there but with a some code and a comment saying \"  // we have our reasons \"\r\nShould this remove it to?"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11728",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11728/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11728/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11728/events",
    "html_url": "https://github.com/npm/npm/issues/11728",
    "id": 137479185,
    "number": 11728,
    "title": "urgent, the project suddenly turn to broken",
    "user": {
      "login": "wahengchang",
      "id": 5538753,
      "avatar_url": "https://avatars.githubusercontent.com/u/5538753?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/wahengchang",
      "html_url": "https://github.com/wahengchang",
      "followers_url": "https://api.github.com/users/wahengchang/followers",
      "following_url": "https://api.github.com/users/wahengchang/following{/other_user}",
      "gists_url": "https://api.github.com/users/wahengchang/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/wahengchang/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/wahengchang/subscriptions",
      "organizations_url": "https://api.github.com/users/wahengchang/orgs",
      "repos_url": "https://api.github.com/users/wahengchang/repos",
      "events_url": "https://api.github.com/users/wahengchang/events{/privacy}",
      "received_events_url": "https://api.github.com/users/wahengchang/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T05:58:09Z",
    "updated_at": "2016-03-01T05:58:09Z",
    "closed_at": null,
    "body": "our product was working very well on this morning, after a break, when I started running it and the terminal showed the msg below, please help\r\n\r\nnpm ERR! Darwin 14.5.0\r\nnpm ERR! argv \"/usr/local/bin/node\" \"/usr/local/bin/npm\" \"update\" \"npm\" \"-g\"\r\nnpm ERR! node v5.0.0\r\nnpm ERR! npm  v3.3.9\r\nnpm ERR! code EMISSINGARG\r\n\r\n\r\n0 info it worked if it ends with ok\r\n1 verbose cli [ '/usr/local/bin/node',\r\n1 verbose cli   '/usr/local/bin/npm',\r\n1 verbose cli   'update',\r\n1 verbose cli   'npm',\r\n1 verbose cli   '-g' ]\r\n2 info using npm@3.3.9\r\n3 info using node@v5.0.0\r\n4 verbose stack Error: Missing required argument #1\r\n4 verbose stack     at module.exports (/usr/local/lib/node_modules/npm/lib/install/mutate-into-logical-tree.js:12:3)\r\n4 verbose stack     at /usr/local/lib/node_modules/npm/lib/outdated.js:78:5\r\n4 verbose stack     at /usr/local/lib/node_modules/npm/lib/outdated.js:62:20\r\n4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-tree/rpt.js:186:9\r\n4 verbose stack     at f (/usr/local/lib/node_modules/npm/node_modules/once/once.js:17:25)\r\n4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-tree/rpt.js:152:20\r\n4 verbose stack     at f (/usr/local/lib/node_modules/npm/node_modules/once/once.js:17:25)\r\n4 verbose stack     at then (/usr/local/lib/node_modules/npm/node_modules/read-package-tree/rpt.js:121:22)\r\n4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-tree/rpt.js:81:20\r\n4 verbose stack     at fs.js:1657:23\r\n5 verbose cwd /Users/staging/Desktop/sourcetree/iStagingHomeBPIntegrate\r\n6 error Darwin 14.5.0\r\n7 error argv \"/usr/local/bin/node\" \"/usr/local/bin/npm\" \"update\" \"npm\" \"-g\"\r\n8 error node v5.0.0\r\n9 error npm  v3.3.9\r\n10 error code EMISSINGARG\r\n11 error typeerror Error: Missing required argument #1\r\n11 error typeerror     at module.exports (/usr/local/lib/node_modules/npm/lib/install/mutate-into-logical-tree.js:12:3)\r\n11 error typeerror     at /usr/local/lib/node_modules/npm/lib/outdated.js:78:5\r\n11 error typeerror     at /usr/local/lib/node_modules/npm/lib/outdated.js:62:20\r\n11 error typeerror     at /usr/local/lib/node_modules/npm/node_modules/read-package-tree/rpt.js:186:9\r\n11 error typeerror     at f (/usr/local/lib/node_modules/npm/node_modules/once/once.js:17:25)\r\n11 error typeerror     at /usr/local/lib/node_modules/npm/node_modules/read-package-tree/rpt.js:152:20\r\n11 error typeerror     at f (/usr/local/lib/node_modules/npm/node_modules/once/once.js:17:25)\r\n11 error typeerror     at then (/usr/local/lib/node_modules/npm/node_modules/read-package-tree/rpt.js:121:22)\r\n11 error typeerror     at /usr/local/lib/node_modules/npm/node_modules/read-package-tree/rpt.js:81:20\r\n11 error typeerror     at fs.js:1657:23\r\n11 error typeerror This is an error with npm itself. Please report this error at:\r\n11 error typeerror     <http://github.com/npm/npm/issues>\r\n12 verbose exit [ 1, true ]\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11727",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11727/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11727/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11727/events",
    "html_url": "https://github.com/npm/npm/issues/11727",
    "id": 137474757,
    "number": 11727,
    "title": "Refusing to install parse-server as a dependency of itself",
    "user": {
      "login": "bhushanawasare90",
      "id": 16936524,
      "avatar_url": "https://avatars.githubusercontent.com/u/16936524?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/bhushanawasare90",
      "html_url": "https://github.com/bhushanawasare90",
      "followers_url": "https://api.github.com/users/bhushanawasare90/followers",
      "following_url": "https://api.github.com/users/bhushanawasare90/following{/other_user}",
      "gists_url": "https://api.github.com/users/bhushanawasare90/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/bhushanawasare90/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/bhushanawasare90/subscriptions",
      "organizations_url": "https://api.github.com/users/bhushanawasare90/orgs",
      "repos_url": "https://api.github.com/users/bhushanawasare90/repos",
      "events_url": "https://api.github.com/users/bhushanawasare90/events{/privacy}",
      "received_events_url": "https://api.github.com/users/bhushanawasare90/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T05:26:15Z",
    "updated_at": "2016-03-01T05:26:15Z",
    "closed_at": null,
    "body": "I am trying to install 'parse-server' module using command:\r\n**sudo npm install parse-server**\r\nBut, this command is not installing parse-server module properly & giving me an error like:\r\n_\"Refusing to install parse-server as a dependency of itself\"_.\r\nI would like to listen if I am missing something.\r\nFollowing is the screenshot of the Terminal:\r\n![parse-server](https://cloud.githubusercontent.com/assets/16936524/13418519/19e68a1c-df9c-11e5-9b13-8d54166078f8.png)\r\n\r\nThanks in advance."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11726",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11726/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11726/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11726/events",
    "html_url": "https://github.com/npm/npm/issues/11726",
    "id": 137462458,
    "number": 11726,
    "title": "`--depth Infinity` has no effect on `npm update`",
    "user": {
      "login": "GriffinSchneider",
      "id": 999346,
      "avatar_url": "https://avatars.githubusercontent.com/u/999346?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/GriffinSchneider",
      "html_url": "https://github.com/GriffinSchneider",
      "followers_url": "https://api.github.com/users/GriffinSchneider/followers",
      "following_url": "https://api.github.com/users/GriffinSchneider/following{/other_user}",
      "gists_url": "https://api.github.com/users/GriffinSchneider/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/GriffinSchneider/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/GriffinSchneider/subscriptions",
      "organizations_url": "https://api.github.com/users/GriffinSchneider/orgs",
      "repos_url": "https://api.github.com/users/GriffinSchneider/repos",
      "events_url": "https://api.github.com/users/GriffinSchneider/events{/privacy}",
      "received_events_url": "https://api.github.com/users/GriffinSchneider/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T04:06:33Z",
    "updated_at": "2016-03-01T04:16:31Z",
    "closed_at": null,
    "body": "I have a package `package-a`, which depends on `package-b`, which depends on `package-c`.\r\n\r\nI'm working in `package-a`, and all the commands here are run from its root directory (with its `package.json`). \r\nFor this example, lets say that `package-b` depends on `\"package-c\": \"^1.0.0\"`. I have `v1.0.0` of `package-c` currently installed in `node_modules/`, and there's a `v1.0.1` of `package-c` published to npm.\r\n\r\nBased on the docs (https://docs.npmjs.com/cli/update), I would expect `npm --depth Infinity update` to install `package-c@1.0.1`, but it doesn't. After running `npm --depth Infinity update`, I am left with `package-c@1.0.0` installed in my `node_modules` directory.\r\n\r\nOther things I've tried:\r\n- `npm update` does not install a new version of `package-c`. This is what I expect based on the docs.\r\n- `npm --depth 9999 update` _does_ install the new version of `package-c`, which is also what I'd expect. I see that the docs were changed in https://github.com/npm/npm/pull/11150 to recommend `--depth Infinity` instead of `--depth 9999`, but as mentioned above `--depth Infinity` does not seem to work.\r\n\r\nTested on `npm@3.8.0`."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11725",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11725/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11725/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11725/events",
    "html_url": "https://github.com/npm/npm/issues/11725",
    "id": 137438339,
    "number": 11725,
    "title": "Raspberry Pi 2 npm error",
    "user": {
      "login": "rsokolski",
      "id": 16902152,
      "avatar_url": "https://avatars.githubusercontent.com/u/16902152?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/rsokolski",
      "html_url": "https://github.com/rsokolski",
      "followers_url": "https://api.github.com/users/rsokolski/followers",
      "following_url": "https://api.github.com/users/rsokolski/following{/other_user}",
      "gists_url": "https://api.github.com/users/rsokolski/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/rsokolski/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/rsokolski/subscriptions",
      "organizations_url": "https://api.github.com/users/rsokolski/orgs",
      "repos_url": "https://api.github.com/users/rsokolski/repos",
      "events_url": "https://api.github.com/users/rsokolski/events{/privacy}",
      "received_events_url": "https://api.github.com/users/rsokolski/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T01:45:12Z",
    "updated_at": "2016-03-01T01:45:12Z",
    "closed_at": null,
    "body": "Any help for me :)???\r\n\r\npi@raspberrypi:~/alexaHarmonyApp-master $ npm install\r\nnpm WARN package.json alexa_harmony_hub_app@1.0.0 No repository field.\r\nnpm WARN engine harmonyhubjs-client@1.1.8: wanted: {\"npm\":\">=3.0.0\"} (current: {\"node\":\"0.10.29\",\"npm\":\"1.4.21\"})\r\nnpm ERR! not a package /tmp/npm-10539-NwYMexze/github.com/swissmanu/ltx.git\r\nnpm ERR! Error: ENOENT, open '/tmp/npm-10539-NwYMexze/github.com/swissmanu/ltx.git-unpack/package.json'\r\nnpm ERR! If you need help, you may report this *entire* log,\r\nnpm ERR! including the npm and node versions, at:\r\nnpm ERR!     <http://github.com/npm/npm/issues>\r\n\r\nnpm ERR! System Linux 4.1.17-v7+\r\nnpm ERR! command \"/usr/bin/nodejs\" \"/usr/bin/npm\" \"install\"\r\nnpm ERR! cwd /home/pi/alexaHarmonyApp-master\r\nnpm ERR! node -v v0.10.29\r\nnpm ERR! npm -v 1.4.21\r\nnpm ERR! path /tmp/npm-10539-NwYMexze/github.com/swissmanu/ltx.git-unpack/package.json\r\nnpm ERR! code ENOENT\r\nnpm ERR! errno 34\r\nnpm ERR! \r\nnpm ERR! Additional logging details can be found in:\r\nnpm ERR!     /home/pi/alexaHarmonyApp-master/npm-debug.log\r\nnpm ERR! not ok code 0\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11724",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11724/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11724/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11724/events",
    "html_url": "https://github.com/npm/npm/issues/11724",
    "id": 137423915,
    "number": 11724,
    "title": "Feature request: packages naming",
    "user": {
      "login": "delvedor",
      "id": 4865608,
      "avatar_url": "https://avatars.githubusercontent.com/u/4865608?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/delvedor",
      "html_url": "https://github.com/delvedor",
      "followers_url": "https://api.github.com/users/delvedor/followers",
      "following_url": "https://api.github.com/users/delvedor/following{/other_user}",
      "gists_url": "https://api.github.com/users/delvedor/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/delvedor/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/delvedor/subscriptions",
      "organizations_url": "https://api.github.com/users/delvedor/orgs",
      "repos_url": "https://api.github.com/users/delvedor/repos",
      "events_url": "https://api.github.com/users/delvedor/events{/privacy}",
      "received_events_url": "https://api.github.com/users/delvedor/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [
      {
        "url": "https://api.github.com/repos/npm/npm/labels/feature-request",
        "name": "feature-request",
        "color": "98DCFF"
      }
    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-03-01T00:26:22Z",
    "updated_at": "2016-03-01T00:53:04Z",
    "closed_at": null,
    "body": "Hi all, I've tried to search in the issues, but I don't know how call this \"feature\".\r\n\r\nA lot of little modules approach is better than some big modules approach, but the problem of the *\"little modules approach\"* is that you need to nominate these, and you will need **a lot** of names!\r\nSo, I'm not a god of math, but is very easy that two people want to give the **same name** to different packages.\r\n\r\nSo my question is:\r\nWhy npm don't have / don't adopt a different packages naming?\r\nA good example could be **username/package**.\r\nSo after publish a package named *foo*, I can install it by typing\r\n```\r\nnpm install delvedor/foo\r\n```\r\nor if I want the *otherUser* foo package\r\n```\r\nnpm install otheUser/foo\r\n```\r\n\r\nIt was already proposed?\r\n- If not, do you think could be a good idea? Why?\r\n- If yes, where?\r\n\r\nThanks."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11723",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11723/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11723/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11723/events",
    "html_url": "https://github.com/npm/npm/issues/11723",
    "id": 137416536,
    "number": 11723,
    "title": "error npm install on Win 7 64 BIT",
    "user": {
      "login": "ebaad",
      "id": 8718000,
      "avatar_url": "https://avatars.githubusercontent.com/u/8718000?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/ebaad",
      "html_url": "https://github.com/ebaad",
      "followers_url": "https://api.github.com/users/ebaad/followers",
      "following_url": "https://api.github.com/users/ebaad/following{/other_user}",
      "gists_url": "https://api.github.com/users/ebaad/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ebaad/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ebaad/subscriptions",
      "organizations_url": "https://api.github.com/users/ebaad/orgs",
      "repos_url": "https://api.github.com/users/ebaad/repos",
      "events_url": "https://api.github.com/users/ebaad/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ebaad/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-02-29T23:48:57Z",
    "updated_at": "2016-02-29T23:48:57Z",
    "closed_at": null,
    "body": "\r\n[npm-debug.txt](https://github.com/npm/npm/files/151949/npm-debug.txt)\r\nTried to install ember after a nodejs V 5.7.0 stable install and got the following error, any help would be appreciated.\r\n\r\n\r\n> npm ERR! Windows_NT 6.1.7601\r\n> npm ERR! argv \"C:\\\\Program Files\\\\nodejs\\\\node.exe\" \"C:\\\\Program Files\\\\nodejs\\\\node_modules\\\\npm\\\\bin\\\\npm-cli.js\" \"install\" \"-g\"\r\n>  \"ember-cli\"\r\n> npm ERR! node v5.7.0\r\n> npm ERR! npm  v3.6.0\r\n> npm ERR! code SELF_SIGNED_CERT_IN_CHAIN\r\n> \r\n> npm ERR! self signed certificate in certificate chain\r\n> npm ERR!\r\n> npm ERR! If you need help, you may report this error at:\r\n> npm ERR!     <https://github.com/npm/npm/issues>\r\n> \r\n> npm ERR! Please include the following file with any support request:"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11722",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11722/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11722/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11722/events",
    "html_url": "https://github.com/npm/npm/issues/11722",
    "id": 137385975,
    "number": 11722,
    "title": "Package installation via github fails silently if package name doesn't match",
    "user": {
      "login": "apfelbox",
      "id": 1032411,
      "avatar_url": "https://avatars.githubusercontent.com/u/1032411?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/apfelbox",
      "html_url": "https://github.com/apfelbox",
      "followers_url": "https://api.github.com/users/apfelbox/followers",
      "following_url": "https://api.github.com/users/apfelbox/following{/other_user}",
      "gists_url": "https://api.github.com/users/apfelbox/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/apfelbox/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/apfelbox/subscriptions",
      "organizations_url": "https://api.github.com/users/apfelbox/orgs",
      "repos_url": "https://api.github.com/users/apfelbox/repos",
      "events_url": "https://api.github.com/users/apfelbox/events{/privacy}",
      "received_events_url": "https://api.github.com/users/apfelbox/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-02-29T21:45:55Z",
    "updated_at": "2016-02-29T21:45:55Z",
    "closed_at": null,
    "body": "Hi,\r\n\r\nI just wanted to install a package with the following `package.json`:\r\n\r\n```json\r\n{\r\n  \"private\": true,\r\n  \"dependencies\": {\r\n    \"matchMedia\": \"github:paulirish/matchMedia.js\"\r\n  }\r\n}\r\n```\r\n\r\nThis doesn't work, as the package name was changed in the meantime to `matchmedia-polyfill`. But if I run `npm install` with this `package.json`, I just get… nothing.\r\n\r\nNo package, but no error message either. Just nothing:\r\n\r\n\r\n```bash\r\n$ npm install\r\n\r\n$\r\n```\r\n\r\nI would suggest to show an error message in case the package name doesn't match the found `package.json` in the git repository?"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11721",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11721/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11721/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11721/events",
    "html_url": "https://github.com/npm/npm/issues/11721",
    "id": 137359663,
    "number": 11721,
    "title": "npm run fails on Windows when command contains \"./\"",
    "user": {
      "login": "cageygee3",
      "id": 10719780,
      "avatar_url": "https://avatars.githubusercontent.com/u/10719780?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/cageygee3",
      "html_url": "https://github.com/cageygee3",
      "followers_url": "https://api.github.com/users/cageygee3/followers",
      "following_url": "https://api.github.com/users/cageygee3/following{/other_user}",
      "gists_url": "https://api.github.com/users/cageygee3/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/cageygee3/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/cageygee3/subscriptions",
      "organizations_url": "https://api.github.com/users/cageygee3/orgs",
      "repos_url": "https://api.github.com/users/cageygee3/repos",
      "events_url": "https://api.github.com/users/cageygee3/events{/privacy}",
      "received_events_url": "https://api.github.com/users/cageygee3/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-02-29T20:16:04Z",
    "updated_at": "2016-02-29T20:16:04Z",
    "closed_at": null,
    "body": "I'm working on a project with a team of mixed operating systems (Windows, Linux, Mac) and our package.json has a few `npm run` definitions that include the dot-slash notation for executing a binary:\r\n\r\n```\r\n    ...\r\n    \"prod\": \"./scripts/build.sh\",\r\n    \"test\": \"./scripts/test.sh\",\r\n    ...\r\n```\r\n\r\nWhen I try to run these commands in my git bash shell in Windows, I get this error:\r\n\r\n```\r\n$ npm run test\r\n\r\n> ...@0.0.1 test C:\\Users\\...\r\n> ./scripts/test.sh\r\n\r\n'.' is not recognized as an internal or external command,\r\noperable program or batch file.\r\n\r\nnpm ERR! Windows_NT 10.0.10586\r\nnpm ERR! argv \"C:\\\\Program Files\\\\nodejs\\\\node.exe\" \"C:\\\\Program Files\\\\nodejs\\\\node_modules\\\\npm\\\\bin\\\\npm-cli.js\" \"run\" \"test\"\r\nnpm ERR! node v5.5.0\r\nnpm ERR! npm  v3.7.0\r\nnpm ERR! code ELIFECYCLE\r\nnpm ERR! ...@0.0.1 test: `./scripts/test.sh`\r\nnpm ERR! Exit status 1\r\n```\r\n\r\nThe dot-slash notation works just fine when run directly in the shell, so this is an issue with npm."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11720",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11720/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11720/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11720/events",
    "html_url": "https://github.com/npm/npm/issues/11720",
    "id": 137348745,
    "number": 11720,
    "title": "Installation understanding with IONIC framework and cordova",
    "user": {
      "login": "syedfahad",
      "id": 5411955,
      "avatar_url": "https://avatars.githubusercontent.com/u/5411955?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/syedfahad",
      "html_url": "https://github.com/syedfahad",
      "followers_url": "https://api.github.com/users/syedfahad/followers",
      "following_url": "https://api.github.com/users/syedfahad/following{/other_user}",
      "gists_url": "https://api.github.com/users/syedfahad/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/syedfahad/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/syedfahad/subscriptions",
      "organizations_url": "https://api.github.com/users/syedfahad/orgs",
      "repos_url": "https://api.github.com/users/syedfahad/repos",
      "events_url": "https://api.github.com/users/syedfahad/events{/privacy}",
      "received_events_url": "https://api.github.com/users/syedfahad/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-02-29T19:33:12Z",
    "updated_at": "2016-02-29T19:33:12Z",
    "closed_at": null,
    "body": "I have installed  _node.js_ and then applied following command to install **cordova** \r\n\r\n> npm install -g cordova\r\n\r\nthen it shows the errors as in the image\r\n![23123213213213](https://cloud.githubusercontent.com/assets/5411955/13406306/94b645b8-ded7-11e5-9c61-d169b23c42b1.JPG)\r\n\r\n\r\nwhat it's mean, Is it installed successfully or not?\r\nand it shows the **cordova** version _6.0.0_"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11717",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11717/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11717/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11717/events",
    "html_url": "https://github.com/npm/npm/issues/11717",
    "id": 137262923,
    "number": 11717,
    "title": "Error: EPERM, chown '/[WORKING DIR]/node_modules/[PACKAGE NAME]/package.json'",
    "user": {
      "login": "giacecco",
      "id": 303693,
      "avatar_url": "https://avatars.githubusercontent.com/u/303693?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/giacecco",
      "html_url": "https://github.com/giacecco",
      "followers_url": "https://api.github.com/users/giacecco/followers",
      "following_url": "https://api.github.com/users/giacecco/following{/other_user}",
      "gists_url": "https://api.github.com/users/giacecco/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/giacecco/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/giacecco/subscriptions",
      "organizations_url": "https://api.github.com/users/giacecco/orgs",
      "repos_url": "https://api.github.com/users/giacecco/repos",
      "events_url": "https://api.github.com/users/giacecco/events{/privacy}",
      "received_events_url": "https://api.github.com/users/giacecco/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-02-29T13:58:07Z",
    "updated_at": "2016-02-29T13:59:22Z",
    "closed_at": null,
    "body": "This is an old issue that likely was never solved, e.g. in https://github.com/npm/npm/issues/7563 or https://github.com/npm/npm/issues/6905, usually addressed by the very kind @smikes . \r\n\r\nI need to use node 0.12.x and its latest associated npm to be as close as possible to a deployment environment I can't control (Gandi.net's basic hosting). To be able to use an old node, I test my code in a clean Docker container with the latest Debian and node 0.12.x from nodesource. \r\n\r\nThe installation of npm packages however always fails with something that looks like the console output below. Remember that all of this runs as root.\r\n\r\n```\r\nroot@3adc90053544:/opt# npm install\r\nnpm WARN package.json dicoim-main-website@0.0.1 No README data\r\nnpm ERR! tar.unpack untar error /root/.npm/express/4.13.4/package.tgz\r\nnpm ERR! Linux 4.1.18-boot2docker\r\nnpm ERR! argv \"/usr/bin/node\" \"/usr/bin/npm\" \"install\"\r\nnpm ERR! node v0.12.10\r\nnpm ERR! npm  v2.14.9\r\nnpm ERR! path /opt/node_modules/express/package.json\r\nnpm ERR! code EPERM\r\nnpm ERR! errno -1\r\n\r\nnpm ERR! Error: EPERM, chown '/opt/node_modules/express/package.json'\r\nnpm ERR!     at Error (native)\r\nnpm ERR!  { [Error: EPERM, chown '/opt/node_modules/express/package.json']\r\nnpm ERR!   errno: -1,\r\nnpm ERR!   code: 'EPERM',\r\nnpm ERR!   path: '/opt/node_modules/express/package.json',\r\nnpm ERR!   fstream_finish_call: 'chown',\r\nnpm ERR!   fstream_type: 'File',\r\nnpm ERR!   fstream_path: '/opt/node_modules/express/package.json',\r\nnpm ERR!   fstream_class: 'FileWriter',\r\nnpm ERR!   fstream_stack: \r\nnpm ERR!    [ '/usr/lib/node_modules/npm/node_modules/fstream/lib/writer.js:306:21',\r\nnpm ERR!      '/usr/lib/node_modules/npm/node_modules/graceful-fs/polyfills.js:210:7',\r\nnpm ERR!      'FSReqWrap.oncomplete (fs.js:95:15)' ] }\r\nnpm ERR! \r\nnpm ERR! Please try running this command again as root/Administrator.\r\n\r\nnpm ERR! Please include the following file with any support request:\r\nnpm ERR!     /opt/npm-debug.log\r\nroot@3adc90053544:/opt\r\n```\r\n\r\nBecause this is a Debian image prepared for Docker, I'm already running this as root and there's not even a _sudo_ command available. ```npm cache clean``` achieves nothing. The cache is in ```/root/.npm```. The full _npm-debug.log_ is at https://gist.github.com/giacecco/8504d2e8501b59f610d9 .\r\n\r\nI used node in Docker systems before without ever finding this issue, this is why I presume it is related to 0.x versions of node and was solved eventually in later versions. But is there a workaround for older versions?\r\n\r\nThanks."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11713",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11713/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11713/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11713/events",
    "html_url": "https://github.com/npm/npm/issues/11713",
    "id": 137103333,
    "number": 11713,
    "title": "errno=Host is down",
    "user": {
      "login": "ianmaddox",
      "id": 1526438,
      "avatar_url": "https://avatars.githubusercontent.com/u/1526438?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/ianmaddox",
      "html_url": "https://github.com/ianmaddox",
      "followers_url": "https://api.github.com/users/ianmaddox/followers",
      "following_url": "https://api.github.com/users/ianmaddox/following{/other_user}",
      "gists_url": "https://api.github.com/users/ianmaddox/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ianmaddox/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ianmaddox/subscriptions",
      "organizations_url": "https://api.github.com/users/ianmaddox/orgs",
      "repos_url": "https://api.github.com/users/ianmaddox/repos",
      "events_url": "https://api.github.com/users/ianmaddox/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ianmaddox/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-02-28T22:16:12Z",
    "updated_at": "2016-02-28T22:16:34Z",
    "closed_at": null,
    "body": "I keep getting a Host is down error when trying to clone github repos:\r\n\r\n> ~ npm install pcap\r\n> npm ERR! git clone --template=/Users/ian/.npm/_git-remotes/_templates --mirror git://github.com/zerouid/node-socketwatcher.git /Users/ian/.npm/_git-remotes/git-github-com-zerouid-node-socketwatcher-git-0fe8b101: Cloning into bare repository '/Users/ian/.npm/_git-remotes/git-github-com-zerouid-node-socketwatcher-git-0fe8b101'...\r\n> npm ERR! git clone --template=/Users/ian/.npm/_git-remotes/_templates --mirror git://github.com/zerouid/node-socketwatcher.git /Users/ian/.npm/_git-remotes/git-github-com-zerouid-node-socketwatcher-git-0fe8b101: fatal: unable to connect to github.com:\r\n> npm ERR! git clone --template=/Users/ian/.npm/_git-remotes/_templates --mirror git://github.com/zerouid/node-socketwatcher.git /Users/ian/.npm/_git-remotes/git-github-com-zerouid-node-socketwatcher-git-0fe8b101: github.com[0: 192.30.252.131]: errno=Host is down\r\n> npm ERR! Darwin 15.3.0\r\n> npm ERR! argv \"/usr/local/bin/node\" \"/usr/local/bin/npm\" \"install\" \"pcap\"\r\n> npm ERR! node v4.2.1\r\n> npm ERR! npm  v3.6.0\r\n> npm ERR! code 128\r\n> \r\n> npm ERR! Command failed: git clone --template=/Users/ian/.npm/_git-remotes/_templates --mirror git://github.com/zerouid/node-socketwatcher.git /Users/ian/.npm/_git-remotes/git-github-com-zerouid-node-socketwatcher-git-0fe8b101\r\n> npm ERR! Cloning into bare repository '/Users/ian/.npm/_git-remotes/git-github-com-zerouid-node-socketwatcher-git-0fe8b101'...\r\n> npm ERR! fatal: unable to connect to github.com:\r\n> npm ERR! github.com[0: 192.30.252.131]: errno=Host is down\r\n> npm ERR! \r\n> npm ERR! \r\n> npm ERR! \r\n> npm ERR! If you need help, you may report this error at:\r\n> npm ERR!     <https://github.com/npm/npm/issues>"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11712",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11712/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11712/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11712/events",
    "html_url": "https://github.com/npm/npm/issues/11712",
    "id": 137054325,
    "number": 11712,
    "title": "Can't install any npm package",
    "user": {
      "login": "kaanyilgin",
      "id": 8467624,
      "avatar_url": "https://avatars.githubusercontent.com/u/8467624?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/kaanyilgin",
      "html_url": "https://github.com/kaanyilgin",
      "followers_url": "https://api.github.com/users/kaanyilgin/followers",
      "following_url": "https://api.github.com/users/kaanyilgin/following{/other_user}",
      "gists_url": "https://api.github.com/users/kaanyilgin/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/kaanyilgin/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/kaanyilgin/subscriptions",
      "organizations_url": "https://api.github.com/users/kaanyilgin/orgs",
      "repos_url": "https://api.github.com/users/kaanyilgin/repos",
      "events_url": "https://api.github.com/users/kaanyilgin/events{/privacy}",
      "received_events_url": "https://api.github.com/users/kaanyilgin/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 2,
    "created_at": "2016-02-28T13:42:36Z",
    "updated_at": "2016-02-29T17:31:56Z",
    "closed_at": null,
    "body": "Hello,\r\n\r\nI reinstalled node.js today again. But I can't install any nodejs package now. Here is the log text:\r\n\r\n0 info it worked if it ends with ok\r\n1 verbose cli [ 'C:\\\\Program Files\\\\nodejs\\\\node.exe',\r\n1 verbose cli   'C:\\\\Program Files\\\\nodejs\\\\node_modules\\\\npm\\\\bin\\\\npm-cli.js',\r\n1 verbose cli   'install',\r\n1 verbose cli   '-g',\r\n1 verbose cli   'gulp' ]\r\n2 info using npm@3.3.12\r\n3 info using node@v5.4.1\r\n4 silly loadCurrentTree Starting\r\n5 silly install loadCurrentTree\r\n6 silly install readGlobalPackageData\r\n7 silly fetchPackageMetaData gulp\r\n8 silly fetchNamedPackageData gulp\r\n9 silly mapToRegistry name gulp\r\n10 silly mapToRegistry using default registry\r\n11 silly mapToRegistry registry https://registry.npmjs.org/\r\n12 silly mapToRegistry uri https://registry.npmjs.org/gulp\r\n13 verbose request uri https://registry.npmjs.org/gulp\r\n14 verbose request no auth needed\r\n15 info attempt registry request try #1 at 3:36:24 PM\r\n16 verbose request id 861fe3a6dabd4ceb\r\n17 http request GET https://registry.npmjs.org/gulp\r\n18 http 200 https://registry.npmjs.org/gulp\r\n19 verbose headers { server: 'CouchDB/1.5.0 (Erlang OTP/R16B03)',\r\n19 verbose headers   etag: '\"ECQ8GN0CYNK4N824KUAVPU2CX\"',\r\n19 verbose headers   'content-type': 'application/json',\r\n19 verbose headers   'cache-control': 'max-age=300',\r\n19 verbose headers   'content-length': '112716',\r\n19 verbose headers   'accept-ranges': 'bytes',\r\n19 verbose headers   date: 'Sun, 28 Feb 2016 13:36:24 GMT',\r\n19 verbose headers   via: '1.1 varnish',\r\n19 verbose headers   age: '55',\r\n19 verbose headers   connection: 'keep-alive',\r\n19 verbose headers   'x-served-by': 'cache-ams4121-AMS',\r\n19 verbose headers   'x-cache': 'HIT',\r\n19 verbose headers   'x-cache-hits': '1',\r\n19 verbose headers   'x-timer': 'S1456666584.511085,VS0,VE0',\r\n19 verbose headers   vary: 'Accept' }\r\n20 silly get cb [ 200,\r\n20 silly get   { server: 'CouchDB/1.5.0 (Erlang OTP/R16B03)',\r\n20 silly get     etag: '\"ECQ8GN0CYNK4N824KUAVPU2CX\"',\r\n20 silly get     'content-type': 'application/json',\r\n20 silly get     'cache-control': 'max-age=300',\r\n20 silly get     'content-length': '112716',\r\n20 silly get     'accept-ranges': 'bytes',\r\n20 silly get     date: 'Sun, 28 Feb 2016 13:36:24 GMT',\r\n20 silly get     via: '1.1 varnish',\r\n20 silly get     age: '55',\r\n20 silly get     connection: 'keep-alive',\r\n20 silly get     'x-served-by': 'cache-ams4121-AMS',\r\n20 silly get     'x-cache': 'HIT',\r\n20 silly get     'x-cache-hits': '1',\r\n20 silly get     'x-timer': 'S1456666584.511085,VS0,VE0',\r\n20 silly get     vary: 'Accept' } ]\r\n21 verbose get saving gulp to C:\\Users\\Kaan Ali Yılgın\\AppData\\Roaming\\npm-cache\\registry.npmjs.org\\gulp\\.cache.json\r\n22 silly install normalizeTree\r\n23 silly loadCurrentTree Finishing\r\n24 silly loadIdealTree Starting\r\n25 silly install loadIdealTree\r\n26 silly cloneCurrentTree Starting\r\n27 silly install cloneCurrentTreeToIdealTree\r\n28 silly cloneCurrentTree Finishing\r\n29 silly loadShrinkwrap Starting\r\n30 silly install loadShrinkwrap\r\n31 silly loadShrinkwrap Finishing\r\n32 silly loadAllDepsIntoIdealTree Starting\r\n33 silly install loadAllDepsIntoIdealTree\r\n34 silly resolveWithNewModule gulp@3.9.1 checking installable status\r\n35 silly cache add args [ 'gulp', null ]\r\n36 verbose cache add spec gulp\r\n37 silly cache add parsed spec Result {\r\n37 silly cache add   raw: 'gulp',\r\n37 silly cache add   scope: null,\r\n37 silly cache add   name: 'gulp',\r\n37 silly cache add   rawSpec: '',\r\n37 silly cache add   spec: '*',\r\n37 silly cache add   type: 'range' }\r\n38 silly addNamed gulp@*\r\n39 verbose addNamed \"*\" is a valid semver range for gulp\r\n40 silly addNameRange { name: 'gulp', range: '*', hasData: false }\r\n41 silly mapToRegistry name gulp\r\n42 silly mapToRegistry using default registry\r\n43 silly mapToRegistry registry https://registry.npmjs.org/\r\n44 silly mapToRegistry uri https://registry.npmjs.org/gulp\r\n45 verbose addNameRange registry:https://registry.npmjs.org/gulp not in flight; fetching\r\n46 verbose request uri https://registry.npmjs.org/gulp\r\n47 verbose request no auth needed\r\n48 info attempt registry request try #1 at 3:36:25 PM\r\n49 http request GET https://registry.npmjs.org/gulp\r\n50 http 200 https://registry.npmjs.org/gulp\r\n51 verbose headers { server: 'CouchDB/1.5.0 (Erlang OTP/R16B03)',\r\n51 verbose headers   etag: '\"ECQ8GN0CYNK4N824KUAVPU2CX\"',\r\n51 verbose headers   'content-type': 'application/json',\r\n51 verbose headers   'cache-control': 'max-age=300',\r\n51 verbose headers   'content-length': '112716',\r\n51 verbose headers   'accept-ranges': 'bytes',\r\n51 verbose headers   date: 'Sun, 28 Feb 2016 13:36:25 GMT',\r\n51 verbose headers   via: '1.1 varnish',\r\n51 verbose headers   age: '56',\r\n51 verbose headers   connection: 'keep-alive',\r\n51 verbose headers   'x-served-by': 'cache-ams4135-AMS',\r\n51 verbose headers   'x-cache': 'HIT',\r\n51 verbose headers   'x-cache-hits': '1',\r\n51 verbose headers   'x-timer': 'S1456666585.647775,VS0,VE0',\r\n51 verbose headers   vary: 'Accept' }\r\n52 silly get cb [ 200,\r\n52 silly get   { server: 'CouchDB/1.5.0 (Erlang OTP/R16B03)',\r\n52 silly get     etag: '\"ECQ8GN0CYNK4N824KUAVPU2CX\"',\r\n52 silly get     'content-type': 'application/json',\r\n52 silly get     'cache-control': 'max-age=300',\r\n52 silly get     'content-length': '112716',\r\n52 silly get     'accept-ranges': 'bytes',\r\n52 silly get     date: 'Sun, 28 Feb 2016 13:36:25 GMT',\r\n52 silly get     via: '1.1 varnish',\r\n52 silly get     age: '56',\r\n52 silly get     connection: 'keep-alive',\r\n52 silly get     'x-served-by': 'cache-ams4135-AMS',\r\n52 silly get     'x-cache': 'HIT',\r\n52 silly get     'x-cache-hits': '1',\r\n52 silly get     'x-timer': 'S1456666585.647775,VS0,VE0',\r\n52 silly get     vary: 'Accept' } ]\r\n53 verbose get saving gulp to C:\\Users\\Kaan Ali Yılgın\\AppData\\Roaming\\npm-cache\\registry.npmjs.org\\gulp\\.cache.json\r\n54 silly addNameRange number 2 { name: 'gulp', range: '*', hasData: true }\r\n55 silly addNameRange versions [ 'gulp',\r\n55 silly addNameRange   [ '0.0.1',\r\n55 silly addNameRange     '0.0.2',\r\n55 silly addNameRange     '0.0.3',\r\n55 silly addNameRange     '0.0.4',\r\n55 silly addNameRange     '0.0.5',\r\n55 silly addNameRange     '0.0.7',\r\n55 silly addNameRange     '0.0.8',\r\n55 silly addNameRange     '0.0.9',\r\n55 silly addNameRange     '0.1.0',\r\n55 silly addNameRange     '0.2.0',\r\n55 silly addNameRange     '1.0.0',\r\n55 silly addNameRange     '1.1.0',\r\n55 silly addNameRange     '1.2.0',\r\n55 silly addNameRange     '1.2.1',\r\n55 silly addNameRange     '2.0.0',\r\n55 silly addNameRange     '2.0.1',\r\n55 silly addNameRange     '2.1.0',\r\n55 silly addNameRange     '2.2.0',\r\n55 silly addNameRange     '2.3.0',\r\n55 silly addNameRange     '2.4.0',\r\n55 silly addNameRange     '2.4.1',\r\n55 silly addNameRange     '2.6.0',\r\n55 silly addNameRange     '2.6.1',\r\n55 silly addNameRange     '2.7.0',\r\n55 silly addNameRange     '3.0.0',\r\n55 silly addNameRange     '3.1.1',\r\n55 silly addNameRange     '3.1.2',\r\n55 silly addNameRange     '3.1.3',\r\n55 silly addNameRange     '3.1.4',\r\n55 silly addNameRange     '3.2.0',\r\n55 silly addNameRange     '3.2.1',\r\n55 silly addNameRange     '3.2.2',\r\n55 silly addNameRange     '3.2.3',\r\n55 silly addNameRange     '3.2.4',\r\n55 silly addNameRange     '3.2.5',\r\n55 silly addNameRange     '3.3.0',\r\n55 silly addNameRange     '3.3.1',\r\n55 silly addNameRange     '3.3.2',\r\n55 silly addNameRange     '3.3.4',\r\n55 silly addNameRange     '3.4.0',\r\n55 silly addNameRange     '3.5.0',\r\n55 silly addNameRange     '3.5.1',\r\n55 silly addNameRange     '3.5.2',\r\n55 silly addNameRange     '3.5.5',\r\n55 silly addNameRange     '3.5.6',\r\n55 silly addNameRange     '3.6.0',\r\n55 silly addNameRange     '3.6.1',\r\n55 silly addNameRange     '3.6.2',\r\n55 silly addNameRange     '3.7.0',\r\n55 silly addNameRange     '3.8.0',\r\n55 silly addNameRange     '3.8.1',\r\n55 silly addNameRange     '3.8.2',\r\n55 silly addNameRange     '3.8.3',\r\n55 silly addNameRange     '3.8.4',\r\n55 silly addNameRange     '3.8.5',\r\n55 silly addNameRange     '3.8.6',\r\n55 silly addNameRange     '3.8.7',\r\n55 silly addNameRange     '3.8.8',\r\n55 silly addNameRange     '3.8.9',\r\n55 silly addNameRange     '3.8.10',\r\n55 silly addNameRange     '3.8.11',\r\n55 silly addNameRange     '3.9.0',\r\n55 silly addNameRange     '3.9.1' ] ]\r\n56 silly addNamed gulp@3.9.1\r\n57 verbose addNamed \"3.9.1\" is a plain semver version for gulp\r\n58 silly mapToRegistry name gulp\r\n59 silly mapToRegistry using default registry\r\n60 silly mapToRegistry registry https://registry.npmjs.org/\r\n61 silly mapToRegistry uri https://registry.npmjs.org/gulp\r\n62 verbose addRemoteTarball https://registry.npmjs.org/gulp/-/gulp-3.9.1.tgz not in flight; adding\r\n63 verbose addRemoteTarball [ 'https://registry.npmjs.org/gulp/-/gulp-3.9.1.tgz',\r\n63 verbose addRemoteTarball   '571ce45928dd40af6514fc4011866016c13845b4' ]\r\n64 info retry fetch attempt 1 at 3:36:26 PM\r\n65 info attempt registry request try #1 at 3:36:26 PM\r\n66 http fetch GET https://registry.npmjs.org/gulp/-/gulp-3.9.1.tgz\r\n67 http fetch 200 https://registry.npmjs.org/gulp/-/gulp-3.9.1.tgz\r\n68 silly fetchAndShaCheck shasum 571ce45928dd40af6514fc4011866016c13845b4\r\n69 verbose addTmpTarball C:\\Users\\KAANAL~1\\AppData\\Local\\Temp\\npm-4900-bf4f0e55\\registry.npmjs.org\\gulp\\-\\gulp-3.9.1.tgz not in flight; adding\r\n70 verbose addTmpTarball already have metadata; skipping unpack for gulp@3.9.1\r\n71 silly rollbackFailedOptional Starting\r\n72 silly rollbackFailedOptional Finishing\r\n73 silly runTopLevelLifecycles Starting\r\n74 silly runTopLevelLifecycles Finishing\r\n75 silly install printInstalled\r\n76 verbose stack Error: UNKNOWN: unknown error, mkdir 'C:\\Users\\Kaan Ali Yılgın\\AppData\\Roaming\\npm-cache\\gulp\\3.9.1\\package'\r\n76 verbose stack     at Error (native)\r\n77 verbose cwd C:\\....\r\n78 error Windows_NT 6.3.9600\r\n79 error argv \"C:\\\\Program Files\\\\nodejs\\\\node.exe\" \"C:\\\\Program Files\\\\nodejs\\\\node_modules\\\\npm\\\\bin\\\\npm-cli.js\" \"install\" \"-g\" \"gulp\"\r\n80 error node v5.4.1\r\n81 error npm  v3.3.12\r\n82 error path C:\\Users\\Kaan Ali Yılgın\\AppData\\Roaming\\npm-cache\\gulp\\3.9.1\\package\r\n83 error code UNKNOWN\r\n84 error errno -4094\r\n85 error syscall mkdir\r\n86 error UNKNOWN: unknown error, mkdir 'C:\\Users\\Kaan Ali Yılgın\\AppData\\Roaming\\npm-cache\\gulp\\3.9.1\\package'\r\n87 error If you need help, you may report this error at:\r\n87 error     <https://github.com/npm/npm/issues>\r\n88 verbose exit [ -4094, true ]\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11710",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11710/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11710/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11710/events",
    "html_url": "https://github.com/npm/npm/issues/11710",
    "id": 137022803,
    "number": 11710,
    "title": "Error installing meanio.",
    "user": {
      "login": "Waffleboy",
      "id": 12254813,
      "avatar_url": "https://avatars.githubusercontent.com/u/12254813?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/Waffleboy",
      "html_url": "https://github.com/Waffleboy",
      "followers_url": "https://api.github.com/users/Waffleboy/followers",
      "following_url": "https://api.github.com/users/Waffleboy/following{/other_user}",
      "gists_url": "https://api.github.com/users/Waffleboy/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/Waffleboy/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/Waffleboy/subscriptions",
      "organizations_url": "https://api.github.com/users/Waffleboy/orgs",
      "repos_url": "https://api.github.com/users/Waffleboy/repos",
      "events_url": "https://api.github.com/users/Waffleboy/events{/privacy}",
      "received_events_url": "https://api.github.com/users/Waffleboy/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-02-28T07:39:21Z",
    "updated_at": "2016-02-28T07:39:21Z",
    "closed_at": null,
    "body": "Hi, i tried installing meanio using\r\n\r\nnpm install -g meanio\r\n\r\nand\r\n\r\nnpm install -g meanio@latest\r\n\r\nBoth both throw me the error as shown.\r\n\r\n![image](https://cloud.githubusercontent.com/assets/12254813/13378053/2d787020-de31-11e5-995e-204c1624d6a7.png)\r\n\r\nI tried reinstalling node and npm, and now use NVM to update to latest version of node and npm and it still doesnt work.\r\n\r\nAnyone have any ideas? :)\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11708",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11708/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11708/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11708/events",
    "html_url": "https://github.com/npm/npm/issues/11708",
    "id": 136985373,
    "number": 11708,
    "title": "ENOTEMPTY: directory not empty, rename   ->  .DELETE",
    "user": {
      "login": "robie1kenobi",
      "id": 17521015,
      "avatar_url": "https://avatars.githubusercontent.com/u/17521015?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/robie1kenobi",
      "html_url": "https://github.com/robie1kenobi",
      "followers_url": "https://api.github.com/users/robie1kenobi/followers",
      "following_url": "https://api.github.com/users/robie1kenobi/following{/other_user}",
      "gists_url": "https://api.github.com/users/robie1kenobi/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/robie1kenobi/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/robie1kenobi/subscriptions",
      "organizations_url": "https://api.github.com/users/robie1kenobi/orgs",
      "repos_url": "https://api.github.com/users/robie1kenobi/repos",
      "events_url": "https://api.github.com/users/robie1kenobi/events{/privacy}",
      "received_events_url": "https://api.github.com/users/robie1kenobi/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 3,
    "created_at": "2016-02-28T00:16:06Z",
    "updated_at": "2016-03-01T12:02:07Z",
    "closed_at": null,
    "body": "reads similar to #11689 \r\n\r\n\r\n[npm-debug.txt](https://github.com/npm/npm/files/149649/npm-debug.txt)\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11706",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11706/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11706/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11706/events",
    "html_url": "https://github.com/npm/npm/issues/11706",
    "id": 136964765,
    "number": 11706,
    "title": "getting error in npm install in ubuntu 14.04",
    "user": {
      "login": "SilentFlame",
      "id": 9693795,
      "avatar_url": "https://avatars.githubusercontent.com/u/9693795?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/SilentFlame",
      "html_url": "https://github.com/SilentFlame",
      "followers_url": "https://api.github.com/users/SilentFlame/followers",
      "following_url": "https://api.github.com/users/SilentFlame/following{/other_user}",
      "gists_url": "https://api.github.com/users/SilentFlame/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/SilentFlame/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/SilentFlame/subscriptions",
      "organizations_url": "https://api.github.com/users/SilentFlame/orgs",
      "repos_url": "https://api.github.com/users/SilentFlame/repos",
      "events_url": "https://api.github.com/users/SilentFlame/events{/privacy}",
      "received_events_url": "https://api.github.com/users/SilentFlame/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 2,
    "created_at": "2016-02-27T20:40:54Z",
    "updated_at": "2016-02-28T20:56:05Z",
    "closed_at": null,
    "body": "```\r\nspeedy@vinay:~$ npm install\r\nnpm ERR! install Couldn't read dependencies\r\nnpm ERR! Error: ENOENT, open '/home/speedy/package.json'\r\nnpm ERR! If you need help, you may report this log at:\r\nnpm ERR!     <http://github.com/isaacs/npm/issues>\r\nnpm ERR! or email it to:\r\nnpm ERR!     <npm-@googlegroups.com>\r\n\r\nnpm ERR! System Linux 3.19.0-51-generic\r\nnpm ERR! command \"/usr/bin/nodejs\" \"/usr/bin/npm\" \"install\"\r\nnpm ERR! cwd /home/speedy\r\nnpm ERR! node -v v0.10.25\r\nnpm ERR! npm -v 1.3.10\r\nnpm ERR! path /home/speedy/package.json\r\nnpm ERR! code ENOENT\r\nnpm ERR! errno 34\r\nnpm ERR! \r\nnpm ERR! Additional logging details can be found in:\r\nnpm ERR!     /home/speedy/npm-debug.log\r\nnpm ERR! not ok code 0\r\n```\r\n\r\nHelp needed."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11705",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11705/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11705/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11705/events",
    "html_url": "https://github.com/npm/npm/issues/11705",
    "id": 136927150,
    "number": 11705,
    "title": "How to resolve this error??",
    "user": {
      "login": "mueedahmed1",
      "id": 12622228,
      "avatar_url": "https://avatars.githubusercontent.com/u/12622228?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/mueedahmed1",
      "html_url": "https://github.com/mueedahmed1",
      "followers_url": "https://api.github.com/users/mueedahmed1/followers",
      "following_url": "https://api.github.com/users/mueedahmed1/following{/other_user}",
      "gists_url": "https://api.github.com/users/mueedahmed1/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/mueedahmed1/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/mueedahmed1/subscriptions",
      "organizations_url": "https://api.github.com/users/mueedahmed1/orgs",
      "repos_url": "https://api.github.com/users/mueedahmed1/repos",
      "events_url": "https://api.github.com/users/mueedahmed1/events{/privacy}",
      "received_events_url": "https://api.github.com/users/mueedahmed1/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 1,
    "created_at": "2016-02-27T14:48:30Z",
    "updated_at": "2016-02-28T02:26:47Z",
    "closed_at": null,
    "body": "![untitled](https://cloud.githubusercontent.com/assets/12622228/13373487/10672c3e-dd8b-11e5-92f5-81468dc87e5e.png)\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11704",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11704/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11704/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11704/events",
    "html_url": "https://github.com/npm/npm/issues/11704",
    "id": 136885903,
    "number": 11704,
    "title": "npm install -g git+ssh://.... problem",
    "user": {
      "login": "chrisjshull",
      "id": 877679,
      "avatar_url": "https://avatars.githubusercontent.com/u/877679?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/chrisjshull",
      "html_url": "https://github.com/chrisjshull",
      "followers_url": "https://api.github.com/users/chrisjshull/followers",
      "following_url": "https://api.github.com/users/chrisjshull/following{/other_user}",
      "gists_url": "https://api.github.com/users/chrisjshull/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/chrisjshull/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/chrisjshull/subscriptions",
      "organizations_url": "https://api.github.com/users/chrisjshull/orgs",
      "repos_url": "https://api.github.com/users/chrisjshull/repos",
      "events_url": "https://api.github.com/users/chrisjshull/events{/privacy}",
      "received_events_url": "https://api.github.com/users/chrisjshull/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 1,
    "created_at": "2016-02-27T06:30:59Z",
    "updated_at": "2016-02-27T18:14:38Z",
    "closed_at": null,
    "body": "This seems to be a variant of https://github.com/npm/npm/issues/9679 but with global installs.\r\n\r\nRun `npm install -g git+ssh://....`.\r\n\r\nThen run `npm -g outd`.\r\n\r\nFails with `Unsupported URL Type: ssh://...`\r\n\r\nDoes not repro with non-global installs.\r\n\r\nLog (redacted):\r\n```\r\n40 verbose stack Error: Unsupported URL Type: ssh://...\r\n40 verbose stack     at parseUrl (/usr/local/lib/node_modules/npm/node_modules/npm-package-arg/npa.js:169:13)\r\n40 verbose stack     at npa (/usr/local/lib/node_modules/npm/node_modules/npm-package-arg/npa.js:75:12)\r\n40 verbose stack     at shouldUpdate (/usr/local/lib/node_modules/npm/lib/outdated.js:335:16)\r\n40 verbose stack     at /usr/local/lib/node_modules/npm/lib/outdated.js:302:23\r\n40 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/slide/lib/async-map.js:52:35\r\n40 verbose stack     at Array.forEach (native)\r\n40 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/slide/lib/async-map.js:52:11\r\n40 verbose stack     at Array.forEach (native)\r\n40 verbose stack     at asyncMap (/usr/local/lib/node_modules/npm/node_modules/slide/lib/async-map.js:51:8)\r\n40 verbose stack     at outdated_ (/usr/local/lib/node_modules/npm/lib/outdated.js:295:3)\r\n...\r\n42 error Darwin 15.4.0\r\n43 error argv \"/usr/local/bin/node\" \"/usr/local/bin/npm\" \"-g\" \"outd\"\r\n44 error node v5.6.0\r\n45 error npm  v3.8.0 \r\n```\r\n"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11703",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11703/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11703/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11703/events",
    "html_url": "https://github.com/npm/npm/pull/11703",
    "id": 136829169,
    "number": 11703,
    "title": "ls: Do not set process.exitCode when ls is executed during the unbuild phase",
    "user": {
      "login": "ixalon",
      "id": 206188,
      "avatar_url": "https://avatars.githubusercontent.com/u/206188?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/ixalon",
      "html_url": "https://github.com/ixalon",
      "followers_url": "https://api.github.com/users/ixalon/followers",
      "following_url": "https://api.github.com/users/ixalon/following{/other_user}",
      "gists_url": "https://api.github.com/users/ixalon/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ixalon/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ixalon/subscriptions",
      "organizations_url": "https://api.github.com/users/ixalon/orgs",
      "repos_url": "https://api.github.com/users/ixalon/repos",
      "events_url": "https://api.github.com/users/ixalon/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ixalon/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [
      {
        "url": "https://api.github.com/repos/npm/npm/labels/review",
        "name": "review",
        "color": "343434"
      }
    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-02-26T22:11:31Z",
    "updated_at": "2016-02-26T22:35:19Z",
    "closed_at": null,
    "pull_request": {
      "url": "https://api.github.com/repos/npm/npm/pulls/11703",
      "html_url": "https://github.com/npm/npm/pull/11703",
      "diff_url": "https://github.com/npm/npm/pull/11703.diff",
      "patch_url": "https://github.com/npm/npm/pull/11703.patch"
    },
    "body": "Prevents invalid error status when `npm install --production <module>` is used.\r\nFixes: #11699\r\n\r\nMy thinking behind this is that `ls` is purely informational when executed as part of an `npm install` (in the unbuild phase). It shouldn't be responsible for deciding if the install should fail (i.e. shouldn't set the process exit code).\r\n\r\nWhen you execute it via `npm ls <package>` (etc.) then it's valid that it may exit with a failure code if `<package>` is not found (the existing behaviour)."
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11702",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11702/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11702/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11702/events",
    "html_url": "https://github.com/npm/npm/issues/11702",
    "id": 136821284,
    "number": 11702,
    "title": "Maximum call stack size exceeded",
    "user": {
      "login": "barthr",
      "id": 9822478,
      "avatar_url": "https://avatars.githubusercontent.com/u/9822478?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/barthr",
      "html_url": "https://github.com/barthr",
      "followers_url": "https://api.github.com/users/barthr/followers",
      "following_url": "https://api.github.com/users/barthr/following{/other_user}",
      "gists_url": "https://api.github.com/users/barthr/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/barthr/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/barthr/subscriptions",
      "organizations_url": "https://api.github.com/users/barthr/orgs",
      "repos_url": "https://api.github.com/users/barthr/repos",
      "events_url": "https://api.github.com/users/barthr/events{/privacy}",
      "received_events_url": "https://api.github.com/users/barthr/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [
      {
        "url": "https://api.github.com/repos/npm/npm/labels/big-bug",
        "name": "big-bug",
        "color": "E03C28"
      },
      {
        "url": "https://api.github.com/repos/npm/npm/labels/bug",
        "name": "bug",
        "color": "A328B3"
      },
      {
        "url": "https://api.github.com/repos/npm/npm/labels/support",
        "name": "support",
        "color": "000000"
      },
      {
        "url": "https://api.github.com/repos/npm/npm/labels/windows",
        "name": "windows",
        "color": "00177D"
      }
    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 3,
    "created_at": "2016-02-26T21:39:48Z",
    "updated_at": "2016-02-27T14:17:27Z",
    "closed_at": null,
    "body": "I was trying to install ansi-to-html package with NPM when some errors occured. So I looked up the dependencies and tried installing them one by one.\r\n\r\nI'm getting this error when running npm install\r\n\r\n```\r\n2376 error tar.unpack untar error C:\\Users\\Bart\\AppData\\Roaming\\npm-cache\\karg\\0.2.1\\package.tgz\r\n2377 warn install:karg@0.2.1 ENOENT: no such file or directory, open 'C:\\Users\\Bart\\node_modules\\.staging\\karg-55affa6e95ec5dee8a277621dc8a228a\\Icon\r\n'\r\n2378 verbose install:karg@0.2.1 Error: ENOENT: no such file or directory, open 'C:\\Users\\Bart\\node_modules\\.staging\\karg-55affa6e95ec5dee8a277621dc8a228a\\Icon\r\n'\r\n2378 verbose install:karg@0.2.1     at Error (native)\r\n2379 silly gunzTarPerm extractEntry lib/js-yaml.js\r\n2380 silly gunzTarPerm modified mode [ 'lib/js-yaml.js', 436, 438 ]\r\n2381 silly gunzTarPerm extractEntry lib/js-yaml/common.js\r\n2382 silly gunzTarPerm modified mode [ 'lib/js-yaml/common.js', 436, 438 ]\r\n2383 silly gunzTarPerm extractEntry lib/argument_parser.js\r\n2384 silly gunzTarPerm extractEntry lib/const.js\r\n2385 error tar.unpack untar error C:\\Users\\Bart\\AppData\\Roaming\\npm-cache\\karg\\1.0.3\\package.tgz\r\n2386 verbose stack RangeError: Maximum call stack size exceeded\r\n2386 verbose stack     at C:\\Program Files\\nodejs\\node_modules\\npm\\lib\\install\\actions.js:54:33\r\n2386 verbose stack     at Array.forEach (native)\r\n2386 verbose stack     at markAsFailed (C:\\Program Files\\nodejs\\node_modules\\npm\\lib\\install\\actions.js:54:16)\r\n2386 verbose stack     at C:\\Program Files\\nodejs\\node_modules\\npm\\lib\\install\\actions.js:57:7\r\n2386 verbose stack     at Array.forEach (native)\r\n2386 verbose stack     at markAsFailed (C:\\Program Files\\nodejs\\node_modules\\npm\\lib\\install\\actions.js:54:16)\r\n2386 verbose stack     at C:\\Program Files\\nodejs\\node_modules\\npm\\lib\\install\\actions.js:57:7\r\n2386 verbose stack     at Array.forEach (native)\r\n2386 verbose stack     at markAsFailed (C:\\Program Files\\nodejs\\node_modules\\npm\\lib\\install\\actions.js:54:16)\r\n2386 verbose stack     at C:\\Program Files\\nodejs\\node_modules\\npm\\lib\\install\\actions.js:57:7\r\n2387 verbose cwd C:\\Users\\Bart\r\n2388 error Windows_NT 10.0.10586\r\n2389 error argv \"C:\\\\Program Files\\\\nodejs\\\\node.exe\" \"C:\\\\Program Files\\\\nodejs\\\\node_modules\\\\npm\\\\bin\\\\npm-cli.js\" \"install\" \"--no-bin-link\" \"--no-color\" \"--no-progress\" \"ansi-to-html\"\r\n2390 error node v5.3.0\r\n2391 error npm  v3.3.12\r\n2392 error Maximum call stack size exceeded\r\n2393 error If you need help, you may report this error at:\r\n2393 error     <https://github.com/npm/npm/issues>\r\n2394 verbose exit [ 1, true ]\r\n```"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11699",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11699/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11699/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11699/events",
    "html_url": "https://github.com/npm/npm/issues/11699",
    "id": 136779476,
    "number": 11699,
    "title": "npm install -production failing since 3.7.4 for all modules (OSX and Linux)",
    "user": {
      "login": "ixalon",
      "id": 206188,
      "avatar_url": "https://avatars.githubusercontent.com/u/206188?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/ixalon",
      "html_url": "https://github.com/ixalon",
      "followers_url": "https://api.github.com/users/ixalon/followers",
      "following_url": "https://api.github.com/users/ixalon/following{/other_user}",
      "gists_url": "https://api.github.com/users/ixalon/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/ixalon/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/ixalon/subscriptions",
      "organizations_url": "https://api.github.com/users/ixalon/orgs",
      "repos_url": "https://api.github.com/users/ixalon/repos",
      "events_url": "https://api.github.com/users/ixalon/events{/privacy}",
      "received_events_url": "https://api.github.com/users/ixalon/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 4,
    "created_at": "2016-02-26T18:45:23Z",
    "updated_at": "2016-02-26T21:38:08Z",
    "closed_at": null,
    "body": "I've been having an error when using `npm install` using the `-production` flag since updating npm to 3.7.5, I've rolled back to 3.7.4 and still see the issue (tested on both OSX and Ubuntu). No debug log is written. 3.7.3 is working correctly.\r\n\r\nnpm correctly installs the module but immediately before exiting (with return code 1) logs: `npm ERR! code 1`.\r\n\r\nI've setup my environment and run a test installing lodash (the same occurs with all modules I've tried) as such:\r\n```bash\r\nrm -rf /tmp/npm-test ~/.npm\r\nmkdir /tmp/npm-test\r\ncd /tmp/npm-test\r\nnpm init -y\r\nnpm install -production -verbose lodash 2> install.log\r\n```\r\n\r\nMy `~/.npmrc` contains only:\r\n```\r\n//registry.npmjs.org/:_authToken=<token>\r\n```\r\n\r\nThe log from 3.7.3: https://gist.github.com/ixalon/bbba9547787f4d356f5a\r\nThe log from 3.7.4: https://gist.github.com/ixalon/260011abf364889a2e76\r\n\r\nThe diff between the resulting two logs shows nothing significant other than the final outcome:\r\n```diff\r\ndiff v3.7.3.log v3.7.4.log\r\n8c8\r\n< npm info using npm@3.7.3\r\n---\r\n> npm info using npm@3.7.4\r\n12c12\r\n< npm info attempt registry request try #1 at 6:33:43 PM\r\n---\r\n> npm info attempt registry request try #1 at 6:32:33 PM\r\n14c14\r\n< npm verb request id 41cc55263c6b90a8\r\n---\r\n> npm verb request id 1bc96515d810d857\r\n23c23\r\n< npm verb headers   date: 'Fri, 26 Feb 2016 18:33:45 GMT',\r\n---\r\n> npm verb headers   date: 'Fri, 26 Feb 2016 18:32:34 GMT',\r\n26c26\r\n< npm verb headers   'x-served-by': 'cache-lhr6322-LHR',\r\n---\r\n> npm verb headers   'x-served-by': 'cache-lhr6325-LHR',\r\n29c29\r\n< npm verb headers   'x-timer': 'S1456511624.593941,VS0,VE813',\r\n---\r\n> npm verb headers   'x-timer': 'S1456511554.377427,VS0,VE202',\r\n43,44c43,44\r\n< npm info retry fetch attempt 1 at 6:33:45 PM\r\n< npm info attempt registry request try #1 at 6:33:45 PM\r\n---\r\n> npm info retry fetch attempt 1 at 6:32:34 PM\r\n> npm info attempt registry request try #1 at 6:32:34 PM\r\n48c48\r\n< npm verb addTmpTarball /var/folders/2c/myq6vg0d5rd604gbz7n6_t980000gn/T/npm-60624-1477c735/registry.npmjs.org/lodash/-/lodash-4.5.1.tgz not in flight; adding\r\n---\r\n> npm verb addTmpTarball /var/folders/2c/myq6vg0d5rd604gbz7n6_t980000gn/T/npm-60616-88880e53/registry.npmjs.org/lodash/-/lodash-4.5.1.tgz not in flight; adding\r\n73c73\r\n< npm info ok \r\n---\r\n> npm ERR! code 1\r\n```"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11696",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11696/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11696/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11696/events",
    "html_url": "https://github.com/npm/npm/issues/11696",
    "id": 136630092,
    "number": 11696,
    "title": "npm completion does not work with : ",
    "user": {
      "login": "electricBonfire",
      "id": 1235763,
      "avatar_url": "https://avatars.githubusercontent.com/u/1235763?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/electricBonfire",
      "html_url": "https://github.com/electricBonfire",
      "followers_url": "https://api.github.com/users/electricBonfire/followers",
      "following_url": "https://api.github.com/users/electricBonfire/following{/other_user}",
      "gists_url": "https://api.github.com/users/electricBonfire/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/electricBonfire/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/electricBonfire/subscriptions",
      "organizations_url": "https://api.github.com/users/electricBonfire/orgs",
      "repos_url": "https://api.github.com/users/electricBonfire/repos",
      "events_url": "https://api.github.com/users/electricBonfire/events{/privacy}",
      "received_events_url": "https://api.github.com/users/electricBonfire/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 0,
    "created_at": "2016-02-26T08:37:32Z",
    "updated_at": "2016-02-26T08:37:32Z",
    "closed_at": null,
    "body": "I have the following scripts `copy` `copy:all` `copy:js` `copy:css`\r\n\r\nI have added the completion script.\r\n\r\nIf I type `npm run copy:<tab>` It completes to `npm run copy:copy:`\r\n\r\nAlso if I run `npm run copy:cs<tab>` it completes to `npm run copy:copy:css`"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11695",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11695/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11695/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11695/events",
    "html_url": "https://github.com/npm/npm/issues/11695",
    "id": 136623287,
    "number": 11695,
    "title": "Error happens when I try to run command \"npm install -g generator-webapp\"",
    "user": {
      "login": "AlexCao2016",
      "id": 17490945,
      "avatar_url": "https://avatars.githubusercontent.com/u/17490945?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/AlexCao2016",
      "html_url": "https://github.com/AlexCao2016",
      "followers_url": "https://api.github.com/users/AlexCao2016/followers",
      "following_url": "https://api.github.com/users/AlexCao2016/following{/other_user}",
      "gists_url": "https://api.github.com/users/AlexCao2016/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/AlexCao2016/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/AlexCao2016/subscriptions",
      "organizations_url": "https://api.github.com/users/AlexCao2016/orgs",
      "repos_url": "https://api.github.com/users/AlexCao2016/repos",
      "events_url": "https://api.github.com/users/AlexCao2016/events{/privacy}",
      "received_events_url": "https://api.github.com/users/AlexCao2016/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 1,
    "created_at": "2016-02-26T07:56:16Z",
    "updated_at": "2016-02-26T12:38:34Z",
    "closed_at": null,
    "body": "npm ERR! Error: No compatible version found: generator-jasmine@'^0.2.1'\r\nnpm ERR! Valid install targets:\r\nnpm ERR! [\"0.1.0\",\"0.1.1\",\"0.1.2\",\"0.1.3\",\"0.2.0\",\"0.2.1\",\"0.2.2\"]\r\nnpm ERR!     at installTargetsError (C:\\nodejs\\node_modules\\npm\\lib\\cache.js:719:10)\r\nnpm ERR!     at C:\\nodejs\\node_modules\\npm\\lib\\cache.js:641:10\r\nnpm ERR!     at saved (C:\\nodejs\\node_modules\\npm\\node_modules\\npm-registry-client\\lib\\get.js:138:7)\r\nnpm ERR!     at Object.oncomplete (fs.js:107:15)\r\nnpm ERR! If you need help, you may report this log at:\r\nnpm ERR!     <http://github.com/isaacs/npm/issues>\r\nnpm ERR! or email it to:\r\nnpm ERR!     <npm-@googlegroups.com>\r\n\r\nnpm ERR! System Windows_NT 6.1.7601\r\nnpm ERR! command \"C:\\\\nodejs\\\\\\\\node.exe\" \"C:\\\\nodejs\\\\node_modules\\\\npm\\\\bin\\\\npm-cli.js\" \"install\" \"-g\" \"generator-webapp\"\r\nnpm ERR! cwd C:\\gds\\drug-management\r\nnpm ERR! node -v v0.10.12\r\nnpm ERR! npm -v 1.2.32\r\nnpm ERR!\r\nnpm ERR! Additional logging details can be found in:\r\nnpm ERR!     C:\\gds\\drug-management\\npm-debug.log\r\nnpm ERR! not ok code 0"
  },
  {
    "url": "https://api.github.com/repos/npm/npm/issues/11694",
    "repository_url": "https://api.github.com/repos/npm/npm",
    "labels_url": "https://api.github.com/repos/npm/npm/issues/11694/labels{/name}",
    "comments_url": "https://api.github.com/repos/npm/npm/issues/11694/comments",
    "events_url": "https://api.github.com/repos/npm/npm/issues/11694/events",
    "html_url": "https://github.com/npm/npm/issues/11694",
    "id": 136622372,
    "number": 11694,
    "title": "Yeoman issue when I install with command \"npm install -g yo\"",
    "user": {
      "login": "AlexCao2016",
      "id": 17490945,
      "avatar_url": "https://avatars.githubusercontent.com/u/17490945?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/AlexCao2016",
      "html_url": "https://github.com/AlexCao2016",
      "followers_url": "https://api.github.com/users/AlexCao2016/followers",
      "following_url": "https://api.github.com/users/AlexCao2016/following{/other_user}",
      "gists_url": "https://api.github.com/users/AlexCao2016/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/AlexCao2016/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/AlexCao2016/subscriptions",
      "organizations_url": "https://api.github.com/users/AlexCao2016/orgs",
      "repos_url": "https://api.github.com/users/AlexCao2016/repos",
      "events_url": "https://api.github.com/users/AlexCao2016/events{/privacy}",
      "received_events_url": "https://api.github.com/users/AlexCao2016/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [

    ],
    "state": "open",
    "locked": false,
    "assignee": null,
    "milestone": null,
    "comments": 1,
    "created_at": "2016-02-26T07:49:58Z",
    "updated_at": "2016-02-26T12:37:12Z",
    "closed_at": null,
    "body": "npm ERR! Error: No compatible version found: fullname@'^2.0.0'\r\nnpm ERR! Valid install targets:\r\nnpm ERR! [\"0.1.0\",\"0.1.1\",\"0.2.0\",\"1.0.0\",\"1.1.0\",\"1.1.1\",\"2.0.0\",\"2.0.1\",\"2.1.0\",\"3.0.0\"]\r\nnpm ERR!     at installTargetsError (C:\\nodejs\\node_modules\\npm\\lib\\cache.js:719:10)\r\nnpm ERR!     at C:\\nodejs\\node_modules\\npm\\lib\\cache.js:641:10\r\nnpm ERR!     at saved (C:\\nodejs\\node_modules\\npm\\node_modules\\npm-registry-client\\lib\\get.js:138:7)\r\nnpm ERR!     at Object.oncomplete (fs.js:107:15)\r\nnpm ERR! If you need help, you may report this log at:\r\nnpm ERR!     <http://github.com/isaacs/npm/issues>\r\nnpm ERR! or email it to:\r\nnpm ERR!     <npm-@googlegroups.com>\r\n\r\nnpm ERR! System Windows_NT 6.1.7601\r\nnpm ERR! command \"C:\\\\nodejs\\\\\\\\node.exe\" \"C:\\\\nodejs\\\\node_modules\\\\npm\\\\bin\\\\npm-cli.js\" \"install\" \"-g\" \"yo\" npm ERR! cwd C:\\gds\\drug-management\r\nnpm ERR! node -v v0.10.12\r\nnpm ERR! npm -v 1.2.32"
  }
]
